src/component/form/TextInputElement/TextInputElement.tsx

```diff
@@ -140,43 +127,35 @@ export class TextInputElement extends React.Component<TextInputProps, TextInputS
         return this.props.focus && this.focusInput();
     }
 
-    public componentDidUpdate(previous: TextInputProps): void {
-        return this.selectedEl && this.selectedEl.scrollIntoView(false);
-    }
+    // public componentDidUpdate(previous: TextInputProps): void {
+    //     return this.selectedEl && this.selectedEl.scrollIntoView(false);
+    // }
 
     private onKeyDown(event: React.KeyboardEvent<HTMLTextElement>): void {
         if (!this.props.autocomplete) {
             return;
         }
 
-        switch (event.keyCode) {
-            case KEY_P:
-                if (!event.ctrlKey) {
-                    break;
-                }
-            case KEY_UP:
+        switch (event.key) {
```

Comment:
Figured it would be a good idea to switch on `event.key` since `event.keyCode` is marked deprecated. What do you think?

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode